meinberlin/apps/contrib/migrations/0001_drop_background_task_table.py

```diff
@@ -0,0 +1,9 @@
+# Generated by Django 3.2.20 on 2023-11-09 07:46
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = []
+
+    operations = [migrations.RunSQL("DROP TABLE IF EXISTS background_task")]
```

Comment:
background tasks creates two tables: `background_task` and `background_task_completedtask`, do you want to keep the second one or is it missing ? :)

Reply:
oh no I don't, I didn't add it, slipped me.